pkg/ast/ast.go

```diff
@@ -1592,10 +1591,10 @@ type AlterColumn struct {
 
 // AlterColumnSet is ALTER COLUMN SET clause in ALTER TABLE.
 //
-//     ALTER COLUMN {{.Name | sql}} SET {{.Options | sql}}
+//     ALTER COLUMN {{.Name | sql}} SET {{.Options | sql}} || {{.DefaultExpr | sql}}
```

Comment:
It should be `{{if .Options}}{{.Options | sql}}{{else}}{{.DefaultExpr | sql}}{{end}}`.

Reply:
Fixed, sorry that I didn't understand the doc syntax exactly. Any guideline for this?

Reply:
Syntax is Go's `text/template`, and the three extra filters `sql`, `sqlOpt`, and `sqlJoin` are defined.

- `sql` calls `SQL` method and returns its result.
- `sqlOpt` is similar to `sql`, but it returns empty string when the input is `nil`.
- `sqlJoin` calls `SQL` method against each element of the input slice, then returns the string joining results with the argument as its separator.